packages/datepicker/src/ux-picker-dialog.ts

```diff
@@ -50,6 +50,7 @@ export class UxPickerDialog {
   public selectDate() {
     if (this.selectedDate != null) {
       this.value = this.selectedDate.toDate();
+      this.closeDialog();
```

Comment:
I felt surprised sometimes too when the dialog wasn't automatically closed. Though I think this needs to be configurable, with the default value is auto close. Thought?

Reply:
Caution, don't be mistaken here. The `selectDate()` is called here from the dialog `OK` button, when we expect the dialog to close. 

The reason why I placed the `closedDialog()` now in this function is because I removed it from a `valueChanged()` in the main `ux-datepicker` VM. When it was in the valueChanged method it used to cause issues when the value was changed (for exemple when it was validated) and then the picker would close without any expectation.

Currently the dialog doesn't close automatically at all. We could add this configurable option in the future, it's a good idea.

Reply:
Here is the `valueChanged()` i'm talking about:  https://github.com/aurelia/ux/pull/232/files/ab4aa794c348ca2220a68330c863fd98d587c467#r375810376

Reply:
thanks for the pointer 👍